dex/testing/polygon/genesis.json

```diff
@@ -0,0 +1,97 @@
+{
+  "config": {
+    "chainId": 90001,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 2722000,
+    "muirGlacierBlock": 2722000,
+    "berlinBlock": 13996000,
+    "londonBlock": 22640000,
+    "bor": {
+      "jaipurBlock": 22770000,
+      "delhiBlock": 29638656,
```

Comment:
Shouldn't these values be 0 for simnet ? Based on cursory study of Bor code, I think setting **londonBlock** to 0 in particular might resolve that **panic** you mentioned above, maybe will try this out myself later.

Reply:
Maybe, but I will test with zero values and see if that's the reason why.

Reply:
Removed those block values and everything seems normal now. Thanks.

<img width="1031" alt="Screenshot 2023-06-18 at 6 20 49 PM" src="https://github.com/decred/dcrdex/assets/57448127/b177df52-03e1-486d-a512-59e2e0d1c548">


<img width="552" alt="Screenshot 2023-06-18 at 6 19 28 PM" src="https://github.com/decred/dcrdex/assets/57448127/88ef2e78-bea4-478c-8a94-85f3ce670acf">